packages/create-plugin/templates/panel/provisioning/dashboards/dashboard.json

```diff
@@ -22,36 +22,6 @@
   "links": [],
   "liveNow": false,
   "panels": [
-    {
```

Comment:
I don't think we need to see the same plugin with different scenario in the dashboard. It is confusing to see the panels one with circle other one without circle(due to no data).